packages/docs/src/pages/getting-started/nuxt.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+  <docs-content :config="configComputed" />
+</template>
+
+<script lang="ts">
+import { Vue, Options } from 'vue-class-component'
```

Comment:
```suggestion
import { defineComponent } from 'vue'
```

Reply:
It is auto generated code. Let's let it be, so we can refactor it globally later. I used `yarn generate:docspage`